Resources/Private/Templates/View3D/Standalone.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dlf Embedded 3D Viewer - <model-viewer></title>
```

Comment:
This isn't valid HTML. It should either be `<model-viewer/>` or `&lt;model-viewer&gt;` depending on if it's meant to be an element or just part of the title string.

Reply:
Good point i changed this.